package/MDAnalysis/analysis/nucleicacids.py

```diff
@@ -159,13 +159,25 @@ class NucPairDist(AnalysisBase):
     .. versionchanged:: 2.7.0
         Added static method :attr:`select_strand_atoms` as a
         helper for selecting atom pairs for distance analysis.
+
+    .. versionchanged:: 2.8.0
+       Enabled **parallel execution** with the ``multiprocessing`` and ``dask``
+       backends; use the new method :meth:`get_supported_backends` to see all
+       supported backends.
+       The 'self._res_array' was modified to 'self.results.distances' to
+       enable the implementation of the parallelization.
```

Comment:
Remove
```suggestion
```
The old `_res_array` was never public so it doesn't have to be mentioned in user-facing docs.